src/components/block/index.ts

```diff
@@ -83,6 +83,15 @@ export enum BlockToolAPI {
   ON_PASTE = 'onPaste',
 }
 
+/**
+ * Available block drop zones position w.r.t. focused block.
+ */
+export enum BlockDropZonePosition {
+  TOP = 'top',
+  BOTTOM = 'bottom',
```

Comment:
```suggestion
  Top = 'top',
  Bottom = 'bottom',
```

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
done